docs/sphinx/source/strictdoc.rst

```diff
@@ -519,6 +519,42 @@ Special feature of ``[COMPOSITE_REQUIREMENT]``: like ``[SECTION]`` element, the
 often, a more basic combination of nested ``[SECTION]`` and ``[REQUIREMENT]``
 elements should do the job.
 
+Include Files
```

Comment:
`Include files` (the project follows European style for the capitalisation of headers)